### DISCUSSION TITLE
taro官网太丑了

### DISCUSSION THREAD

**Discussion by night-peiqi** [NervJS/taro#15347] [Feature Request]
前几年经常使用taro，当时就感觉taro的官网太丑了，今天准备启动一个新项目，在打开taro官网之前我还在期待会不会给我点惊喜，结果打开一看还和三年前一样的丑。

这应该是我见过最丑的前端框架官网了吧，不由得先问一句：身为一个前端框架，taro团队真的没有一个具备美感的前端工程师吗，如果没有我愿尽绵薄之力。

吐槽完了，我去捡粪了...

**Comment by koppthe** [comment]
你说的“丑”指的是首页还是文档？

**Reply by night-peiqi** [reply]
你说的首页是哪里，打开taro不就默认进入文档页了吗
![image](https://github.com/NervJS/taro/assets/32918263/8a15c874-d3a9-4cbb-9897-2a7bc79810cf)

**Comment by night-peiqi** [comment]
不跟远的比，就下面这些哪个不比taro的文档美观
![image](https://github.com/NervJS/taro/assets/32918263/783a3b83-3208-4b63-aaa1-f12fd5a167d0)

**Reply by koppthe** [reply]
你觉得这个网站怎样？
https://docusaurus.io/docs/playground

**Reply by ianzone** [reply]
docusaurus 更好！
